server/api/models/post.ts

```diff
@@ -0,0 +1,22 @@
+import mongoose from "mongoose";
+import sequence from "mongoose-sequence";
+
+const AutoIncrement = sequence(mongoose);
+
+export interface IPostModel extends mongoose.Document {
+  // TODO
+}
+
+const posts = new mongoose.Schema(
+  {
+    id: { type: Number, unique: true },
+    // TODO
+  },
+  {
+    collection: "posts",
+  }
+);
+
+posts.plugin(AutoIncrement, { inc_field: "id" });
```

Comment:
Would this need to be post_id?

Reply:
Good idea! I should make it consistent with `user_id` and `review_id` as well